driver-core/src/main/com/mongodb/ReadPreferenceHedgeOptions.java

```diff
@@ -28,15 +28,15 @@
  */
 @Immutable
 public final class ReadPreferenceHedgeOptions {
-    private final boolean hedgedReads;
+    private final boolean isEnabled;
```

Comment:
The convention in Java is to only use the "is" suffix on the getter method, not the property, so this should just be "enabled"